builtin/providers/azurerm/resource_arm_virtual_machine.go

```diff
@@ -887,14 +887,7 @@ func flattenAzureRmVirtualMachineOsProfile(osProfile *compute.OSProfile) []inter
 	result["computer_name"] = *osProfile.ComputerName
 	result["admin_username"] = *osProfile.AdminUsername
 	if osProfile.CustomData != nil {
-		var data string
-		if isBase64Encoded(*osProfile.CustomData) {
-			decodedData, _ := base64.StdEncoding.DecodeString(*osProfile.CustomData)
-			data = string(decodedData)
-		} else {
-			data = *osProfile.CustomData
-		}
-		result["custom_data"] = data
+		result["custom_data"] = *osProfile.CustomData
```

Comment:
This I'm not sure about. I tried this with b64 and non-b64 encoded data, and it worked both times. I'm not sure what this code was attempting to do?